qiime2/sdk/usage.py

```diff
@@ -1210,6 +1210,123 @@ def factory():
             return artifact
         return self._usage_variable(name, factory, 'artifact')
 
+    def construct_collection(
```

Comment:
I was going to comment that this should perhaps be `init_collection` to match the others, but I realized this doesn't accept a factory, which is why I suspect you didn't name it such. I think that's probably a good reason to leave it as its own thing.